server/lobbyconnection.py

```diff
@@ -888,6 +893,21 @@ def launch_game(self, game, port, is_host=False, use_map=None):
             else:
                 raise ValueError('invalid type argument')
 
+    @asyncio.coroutine
+    async def command_ice_servers(self, message):
+        if self.player:
+
+            ttl = 24 * 3600
```

Comment:
Here's that mysterious hardcoded constant again

Reply:
I don't see an issue with that, it's probably a specified constant in the turn long term cred mechanism rfc, I wouldn't introduce a constant/variable for that as it will never change and its meaning is obvious.

Reply:
I think your comment just demonstrated exactly why it is an issue. No one besides the original author knows what the constant means. It should be extracted to a named constant for readability. Even if it's something like TWILLIO_DEFAULT_TTL. If it is in fact from an RFC, then you could even put a comment with a reference to the RFC at the place where that constant is defined. Also the whole point of constants is that they are... constant, they don't change.

https://www.itjungle.com/2004/08/18/fhg081804-story01/